scenarioo-client/app/step/step.controller.ts

```diff
@@ -51,6 +52,16 @@ function StepController($scope, $routeParams, $location, $route, StepResource, S
         SelectedBranchAndBuildService.callOnSelectionChange(loadStep);
     }
 
+    /**
+     * reload the page to ensure that highlighted changes are properly aligned
+     * if comparison highlights are activated and we are on the comparisons tab.
+     */
+    function refreshIfComparisonActive() {
+        if ($scope.isComparisonChangesHighlighted() && $scope.activeTab === 2) {
+            $route.reload();
```

Comment:
I think it would be better to jump to the comparison page after this in any case and trigger a reload for sure, to give the user some feedback.